src/api/getPet.js

```diff
@@ -0,0 +1,4 @@
+export const getPet = async (index) => {
```

Comment:
Should probably rename the file to `petsApi` or something as we will add delete / edit etc methods here